docs/source/high-level-intro.rst

```diff
@@ -0,0 +1,125 @@
+
+In the execution model for Canton, each party of the application is hosted on a Participant Node (Diagram 1). The Participant Node stores the party’s unique projection and history of the shared system of record. Participant Nodes synchronize by running a consensus protocol (the Canton Protocol) between them. The protocol is executed by sending encrypted messages through Domains, which route messages and offer guaranteed delivery and order consistency. Domains are also units of access control and availability, meaning an application can be additionally protected from interference by other applications or malicious actors by synchronizing it only through a given domain, and restricting which participants can connect to it. 
+
+Diagram 1:
+
+.. figure:: arch-intro-1.png
+   :alt: A Domain (center) with four Participant Nodes. Participant Node One hosts Party A; Participant Node Two hosts Party B; Participant Node Three hosts Party C; and Participant Node Four hosts Parties D, E, and F. The Domain can be centralized or distributed, public or private. 
+
+In a composed solution, each domain is a sub-network. A Participant Node connects to one or more Domains, enabling transactions that span Domains (Diagram 2).
+
+Diagram 2:
+
+.. figure:: arch-intro-2.png
+   :alt: Three Domains with five Participant Nodes, each hosting one or more parties. Domains A (HL Fabric) and B (Ethereum) have two Domain Nodes each, while Domain C (SQL) has a single Domain Node. Each Participant Node can connect to different Domain Nodes across different Domains.
+
+
+Transfer Example Using Daml
+***************************
+
+Consider the transfer example described above with Alice and Bob. Using Daml, the process looks like this:
+#. Alice logs into her online banking at Bank A and enters a transfer to Bob at Bank B.
+#. The online banking backend creates a transaction that deducts $100 from Alice’s account and creates a transfer to Bob at Bank B. 
+#. When Bank B accepts the transfer, Bank A credits $100 to Bank B’s account at Bank A and Bank B simultaneously credits Bob’s account by $100.
+#. Bob’s online banking interfaces with the Daml Ledger and can see the incoming funds in real time.
+
+At every point, ownership of the $100 is completely clear and all systems are fully consistent. 
+
+Next Steps
+**********
+
+The suggested next steps are:
+
+* Learn about the Daml language and the Daml Ledger Model. :doc:`Writing Daml <daml/intro/0_Intro>` will introduce you to the basics of a Daml contract, the Daml Ledger model, and the core features of the Daml language. You’ll notice that testing your contracts, including :doc:`testing for failures <daml/intro/2_DamlScript.html#test-for-failure>`, is presented very early in this introduction. We strongly recommend that you write tests as part of the initial development of every Daml project.
```

Comment:
```suggestion
* Learn about the Daml language and the Daml Ledger Model. :doc:`Writing Daml <daml/intro/0_Intro>` will introduce you to the basics of a Daml contract, the Daml Ledger model, and the core features of the Daml language. You’ll notice that testing your contracts, including :ref:`testing for failures<intro_2_failure>`, is presented very early in this introduction. We strongly recommend that you write tests as part of the initial development of every Daml project.
```

You cannot link to an HTML document because it's the output of the RST and it doesn't exist while you're processing the RST itself.